tests/serialization/test_dag_serialization.py

```diff
@@ -554,6 +554,9 @@ def validate_deserialized_dag(self, serialized_dag, dag):
         compare_serialization_list = {
             "dataset_triggers",
         }
+        compare_serialization_list = {
+            "dataset_triggers",
+        }
```

Comment:
```suggestion
        compare_serialization_list = {
            "dataset_triggers",
        }
```

Reply:
Changed it.I think is was messed during rebase.